apps/notifications/tasks.py

```diff
@@ -109,7 +110,11 @@ class Meta:
         )
 
 
-@celery_app.task(bind=True)
+@celery_app.task(
+    bind=True,
+    autoretry_for=(ObjectDoesNotExist,),
```

Comment:
The actual exception we see is `Notification.DoesNotExist`, will this work for that as well?

Reply:
https://docs.djangoproject.com/en/3.1/ref/exceptions/#objectdoesnotexist
Yeah! It might not need to be this general tho

Reply:
If it works it works 🎉